### PR TITLE
[W5.2][W10-A1]Joshua Lee Kai Sheng

### PR DESCRIPTION
**Pull request by jlks96**
-abstracted out Formatter class from TextUi class

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package seedu.addressbook.ui;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static seedu.addressbook.common.Messages.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:10]: Import the necessary classes explicitly.

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import static seedu.addressbook.common.Messages.MESSAGE_INIT_FAILED;
+
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:16]: You need to move the `DIVIDER` as well to this class to achieve full SRP.

**Comment by okkhoy** [comment]
@jlks96 
Some comments added. Please ack & close the PR after reading comments.

**Comment by jlks96** [comment]
Acknowledged. Thanks.

**Pull request closed without merging**